substrate/frame/benchmarking/src/utils.rs

```diff
@@ -238,7 +259,10 @@ sp_api::decl_runtime_apis! {
 		/// Parameters
 		/// - `extra`: Also list benchmarks marked "extra" which would otherwise not be
 		///            needed for weight calculation.
-		fn benchmark_metadata(extra: bool) -> (Vec<BenchmarkList>, Vec<StorageInfo>, Weight, RuntimeDbWeight);
+		///
+		/// Returns:
+		/// - `BenchmarkInfo` - Complete benchmark metadata available for this runtime.
+		fn benchmark_metadata(extra: bool) -> BenchmarkInfo;
```

Comment:
Normally, we keep the old version of the runtime API around. But given that benchmarks operate on newly build runtimes, i guess its fine.